apps/bfd-server/bfd-server-launcher/pom.xml

```diff
@@ -111,19 +111,6 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <!-- Used in tests to verify that requests work as expected. -->
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpclient</artifactId>
-            <version>${org.apache.httpcomponents.httpclient.version}</version>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <artifactId>commons-logging</artifactId>
-                    <groupId>commons-logging</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
```

Comment:
This dependency is no longer necessary since the AWS SDK also requires this dependency and is needed in compile scope.  Removing this old dependency ensures that the server can use SSM-based configuration.